### PR TITLE
add transfers query

### PR DESCRIPTION
**Pull request by CatalinFaurPaul**
## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
-  Add Transfers Query Graphql

  
## Proposed Changes
- Create transfers graphql query - transfers & transfersCount

## How to test
-  `graphql: true`
-  `indexer-v3: true`
Run query:
```
query{
  transfers(input:{
  }){
    txHash
    gasLimit
    gasPrice
    gasUsed
    miniBlockHash
    nonce
    receiverAddress
    receiverShard
    round
    senderAddress
    senderShard
    signature
    status
    value
    fee
    timestamp
    data
    type
    originalTxHash
  }
}
```

### CODE REVIEW

File: src/endpoints/transactions/entities/transaction.ts [hunk 1]
```diff
@@ -102,9 +106,11 @@ export class Transaction {
   @ApiProperty({ type: ScamInfo, nullable: true })
   scamInfo: ScamInfo | undefined = undefined;
 
+  @Field(() => TransactionType, { description: "Transaction type.", nullable: true })
   @ApiProperty({ enum: TransactionType, nullable: true })
   type: TransactionType | undefined = undefined;
 
+  @Field(() => String, { description: "Original tx hasg for the given transaction.", nullable: true })
```
Comment (AlexandruPislariu) [inline src/endpoints/transactions/entities/transaction.ts:113]: typo here 'hasg'

File: src/graphql/entities/transfers/transfers.input.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { Field, Float, InputType } from "@nestjs/graphql";
+import { SortOrder } from "src/common/entities/sort.order";
+import { TransactionFilter } from "src/endpoints/transactions/entities/transaction.filter";
+import { TransactionStatus } from "src/endpoints/transactions/entities/transaction.status";
+
+@InputType({ description: "Input to retreive the given transfers count for." })
```
Comment (AlexandruPislariu) [inline src/graphql/entities/transfers/transfers.input.ts:6]: typo here 'retreive'

**Review by AlexandruPislariu — CHANGES_REQUESTED** [review]

File: src/graphql/entities/transfers/transfers.input.ts (hunk 2 above)
Comment (CatalinFaurPaul) [reply src/graphql/entities/transfers/transfers.input.ts]: Fixed

File: src/endpoints/transactions/entities/transaction.ts (hunk 1 above)
Comment (CatalinFaurPaul) [reply src/endpoints/transactions/entities/transaction.ts]: Fixed

**Review by tanghel — APPROVED** [review]

**Pull request merged**